src/test/kotlin/solve/parsers/lines/CSVLinesParserTest.kt

```diff
@@ -0,0 +1,62 @@
+package solve.parsers.lines
+
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import solve.parsers.structures.Line
+import java.io.File
+
+internal class CSVLinesParserTest {
+    @Test
+    fun `Parsing a lines CSV file containing data in standard format`(@TempDir tempFolder: File) {
```

Comment:
(**optional**) Fake stream instead of test file might look better

Reply:
> (**optional**) Fake stream instead of test file might look better

I'm not against implementing with fake threads, but why does it might look better? A fake file is a more realistic situation for us, since we pass the path of this file to the `parse(...)` method. Also, the available tool for this is quite simple and it seems that it was created for these purposes